templates/base/footer.tmpl

```diff
@@ -26,9 +26,7 @@
 {{if .RequireMinicolors}}
 	<script src="{{StaticUrlPrefix}}/vendor/plugins/jquery.minicolors/jquery.minicolors.min.js"></script>
 {{end}}
-{{if .RequireU2F}}
```

Comment:
Why this removal here?, ref https://github.com/go-gitea/gitea/pull/11585

Reply:
Until #11585 is merged, this line completely prevents U2F from operating if the user doesn't have TOTP enrollments.